client.py

```diff
@@ -1,8 +1,26 @@
 # _*_ coding utf-8 _*_
+
 import socket
 
-client = socket.socket(socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_TCP)
-client.connect(5000)
 
-msg = u'this is the message to send'
-client.sendall(msg.encode('utf-8'))
+# address = ('127.0.0.1', 5000)
+# client = socket.socket(socket.AF_INET, socket.SOCK_STREAM, socket.IPPROTO_TCP)
+# client.connect(address)
+
+# request = "GET /index.html HTTP/1.1 \r\n"
+
+
+def send():
+    client.sendall(request.encode('utf-8'))
+    data = client.recv(32)
+    client.close()
+    print (data.decode('utf-8'))
+    return (data.decode('utf-8'))
```

Comment:
You've got a lot of assumptions here about what client and request are.  This method only works because both are bound in the namespace where you invoke this function (the `main` block below).  That's not good.  You should pass both as arguments to the send function instead.  That clarifies that this function requires both and uses them both.